parser-typechecker/src/Unison/Codebase/FileCodebase.hs

```diff
@@ -279,7 +285,7 @@ getRootBranch root = do
  where
   go single = case hashFromString single of
     Nothing -> failWith $ CantParseBranchHead single
-    Just h  -> branchFromFiles root (RawHash h)
+    Just h  -> branchFromFiles True root (RawHash h)
```

Comment:
Recommend a self-documenting data type rather than `Bool`

Reply:
This was a good idea and is done, also added some verbiage on why you would do one mode vs the other.